package.json

```diff
@@ -92,6 +92,7 @@
   "eslintConfig": {
     "extends": "./node_modules/kcd-scripts/eslint.js",
     "rules": {
+      "import/no-useless-path-segments": "off",
```

Comment:
Your fix on `eslint-config-kentcdodds` didn't publish (CI failing) so I added an temporary rule until the publish